mlflow/resource_experiment.go

```diff
@@ -0,0 +1,125 @@
+package mlflow
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Experiment defines the parameters that can be set in the resource.
+type Experiment struct {
+	Name             string `json:"name"`
+	ArtifactLocation string `json:"artifact_location,omitempty" tf:"force_new"`
+	Description      string `json:"description,omitempty"`
+}
+
+// ExperimentDto defines the response object from the API
+type ExperimentDto struct {
+	ExperimentId     string `json:"experiment_id"`
+	Name             string `json:"name"`
+	ArtifactLocation string `json:"artifact_location,omitempty"`
+	LifecycleStage   string `json:"lifecycle_stage,omitempty"`
+	LastUpdateTime   int64  `json:"last_update_time,omitempty"`
+	CreationTime     int64  `json:"creation_time,omitempty"`
+}
+
+type ExperimentUpdateDto struct {
+	ExperimentId string `json:"experiment_id"`
+	NewName      string `json:"new_name"`
+}
+
+type ExperimentsDto struct {
```

Comment:
this should be a package private entity and it's more of a container. so rename it: `ExperimentsDto` -> `experimentWrapper`

Reply:
done